client/modules/posts/viewPost/viewPost.js

```diff
@@ -17,30 +25,25 @@ Template.viewPost.events({
 		if(!this.text){
 			return;
 		}
-
-		post = {
-			target: e.currentTarget,
-			text: this.text,
-			markupText: this.text
-		}
-
+		Session.set('activePost', this._id);
 	},
 
 	'mouseleave .post': function(e) {
-		post = {};
+		Session.delete('activePost');
 	},
 
 	'mouseenter .tag a': function(e) {
-		if(!post.text){
+		var activePost = Posts.findOne(Session.get('activePost'));
```

Comment:
Fra http://docs.meteor.com/#/full/mongo_collection:

On the client (and on the server if you specify a connection), a Minimongo instance is created. Minimongo is essentially an in-memory, non-persistent implementation of Mongo in pure JavaScript. It serves as a local cache that stores just the subset of the database that this client is working with. Queries (find) on these collections are served directly out of this cache, without talking to the server.